app/src/org/commcare/android/util/FileUtil.java

```diff
@@ -395,4 +396,20 @@ public static String getDumpDirectory(Context c){
                 return null;
             }
         }
+
+    public static Properties loadProperties(Context c) throws IOException {

```

Comment:
at the moment possibly it would but I think its more generically useful if we need to use other keys or local properties in other areas in the future